tensorboard/plugins/scalar/scalars_plugin_test.py

```diff
@@ -115,29 +116,28 @@ def testRoutesProvided(self):
     self.assertIsInstance(routes['/tags'], collections.Callable)
 
   def generate_run(self, run_name):
-    tf.compat.v1.reset_default_graph()
-    sess = tf.compat.v1.Session()
-    placeholder = tf.compat.v1.placeholder(tf.float32, shape=[3])
-
-    if run_name == self._RUN_WITH_LEGACY_SCALARS:
-      tf.compat.v1.summary.scalar(self._LEGACY_SCALAR_TAG, tf.reduce_mean(input_tensor=placeholder))
-    elif run_name == self._RUN_WITH_SCALARS:
-      summary.op(self._SCALAR_TAG, tf.reduce_sum(input_tensor=placeholder),
-                 display_name=self._DISPLAY_NAME,
-                 description=self._DESCRIPTION)
-    elif run_name == self._RUN_WITH_HISTOGRAM:
-      tf.compat.v1.summary.histogram(self._HISTOGRAM_TAG, placeholder)
-    else:
-      assert False, 'Invalid run name: %r' % run_name
-    summ = tf.compat.v1.summary.merge_all()
-
     subdir = os.path.join(self.logdir, run_name)
     with test_util.FileWriterCache.get(subdir) as writer:
-      writer.add_graph(sess.graph)
       for step in xrange(self._STEPS):
-        feed_dict = {placeholder: [1 + step, 2 + step, 3 + step]}
-        s = sess.run(summ, feed_dict=feed_dict)
-        writer.add_summary(s, global_step=step)
+        placeholder = [1 + step, 2 + step, 3 + step]
```

Comment:
`placeholder` doesn't really make sense as a name for this since it is no longer an actual placeholder

Reply:
Changed to `data`.